Scala/Scala.sublime-syntax

```diff
@@ -92,6 +92,18 @@ contexts:
     - include: late-operators
 
   block-comments:
+    - match: (^\s*)?/\*\*
+      scope: punctuation.definition.comment.scala
+      push:
+        - meta_scope: comment.block.documentation.scala
+        - match: \*/\s*$
```

Comment:
These two PRs are essentially in conflict.  I can just merge them into one if you would like.  They each fix a separate issue with the same functionality.